lib/public/views/Runs/Details/index.js

```diff
@@ -115,6 +131,14 @@ const runDetails = (model) => {
                     h('.w-40.ph1', runDetail(model, data.payload)),
                     model.runs.isEditModeEnabled && editTagsPanel(model),
                 ]),
+                h('.flex-row.w-100.justify-center', [
+                    prevId && h('button.btn.btn-primary.m2#prev-run', {
+                        onclick: () => model.router.go(`/?page=run-detail&id=${prevId}`),
+                    }, 'Prev'),
```

Comment:
I think the logic here should be the other way around. 
`next` for latest runs
`prev` for previous runs
Locally while testing I have seen that `next` took me from Run7 to Run6

Reply:
Prev and next are switched now